lib/Domain/Helper/FilterPhpVersionDirectoryIterator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Phpactor\CodeTransform\Domain\Helper;
+
+use Iterator;
+use FilterIterator;
+use RuntimeException;
+use SplFileInfo;
+
+class FilterPhpVersionDirectoryIterator extends FilterIterator
```

Comment:
should this be here?

Reply:
I might have made a mistake here.
You asked for this logic to not be in the extension, since it was in the codeTransformExtraExtension I put it in this repository.
But the templates are in the code-builder.
So I might just have been confused: I should but in in `code-builder` instead, is that right ?

Reply:
I think this should be a feature of code builder which we integrate with Phpactor -- but the comment is more that this is the nullable types PR, and not related to PHP version template selection?

Reply:
I moved it to  `code-builder` so this PR should be cleaner.